notebooks/lesson02_exploratory-data-analysis.ipynb

```diff
@@ -0,0 +1,2971 @@
+{
```

Comment:
<p>Since this shows california it would be cool to see the actual map next to it or have them look it up and explain the hotspots? just an idea.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lewtun/dslectures/pull/12/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>

Reply:
<p>Great idea - I'll add it in the next commit</p>